src/partitioned-dbs/index.rst

```diff
@@ -105,14 +105,20 @@ sensor readings from an IoT field monitoring service. In this particular
 use case, it's quite logical to group all documents by their ``sensor_id``
 field. In this case, we would call the ``sensor_id`` the partition.
 
+A partition offers two benefits. Firstly, it allows you to simplify the
+workload on the query engine, by allowing the engine to only query shards
+that it needs to query. Secondly, it allows the database to scale in more
+predictable ways.
+
 A good partition has two basic properties. First, it should have a high
 cardinality. That is, a large partitioned database should have many more
-partitions than documents in any single partition. A database that has
-a single partition would be an anti-pattern for this feature. Secondly,
-the amount of data per partition should be "small". The general
-recommendation is to limit individual partitions to less than ten
-gigabytes (10 GB) of data. Which, for the example of sensor documents,
-equates to roughly 60,000 years of data.
+partitions than documents, compared to an unpartitioned database. A database
```

Comment:
Hm, you lose the subtlety here of "significantly more partitions than documents in a single partition." I don't think that's necessarily a good thing.

1,000,000 documents split into 10,000 partitions is 100 docs/partition. That would satisfy the recommendation in the original text, but not your new definition, since 10,000 partitions isn't many more partitions than documents in the original 1,000,000 document database.